modules/EvseManager/EvseManager.cpp

```diff
@@ -774,6 +774,7 @@ void EvseManager::ready() {
         }
     });
 
+    this->charger->signalEvent(types::evse_manager::SessionEventEnum::Enabled);
```

Comment:
This should not be neccessary. When charger->enable() is called it will also emit the event

Reply:
I'll push a commit removing this